addons/a11y/package.json

```diff
@@ -52,6 +52,8 @@
     "core-js": "^3.0.1",
     "global": "^4.3.2",
     "lodash": "^4.17.15",
+    "react": "16.13.1",
```

Comment:
@gaetanmaisse This doesn't seem right to me. If I'm a user writing a third party addon, what version of React should I target? Isn't a peer dependency appropriate for addons where we want it to use whatever version Storybook is using for the manager code?